docs/it-manual/monitoring.md

```diff
@@ -0,0 +1,719 @@
+# Monitoring a CiviForm deployment
+
+CiviForm supports server monitoring via [Prometheus](https://prometheus.io/) metrics visualized in [Grafana](https://grafana.com/).
+
+![](<../.gitbook/assets/monitoring-architecture.png.png>)
+
+## Enabling metrics export
+
+Exporting metrics from the server is optional, and must be enabled by setting the `CIVIFORM_SERVER_METRICS_ENABLED` environment variable to `true`. When enabled, the server exports metrics from the `/metrics` HTTP route. While these metrics data are not sensitive, it is a good practice to prevent access to this route from the public internet (which is done by default when using the [CiviForm terraform deployment system](https://github.com/civiform/cloud-deploy-infra/)).
+
+## Viewing metrics
+
+### AWS
+
+The [CiviForm terraform deployment system](https://github.com/civiform/cloud-deploy-infra/) for AWS deploys the monitoring stack automatically. After deployment, user access to the Grafana dashboard and configuration of the dashboard need to be done manually.
+
+#### Configure access
+
+[AWS Managed Grafana](https://aws.amazon.com/grafana/) uses [AWS IAM Identity Center](https://aws.amazon.com/iam/identity-center/) for access management. **Note that this is a different service from [AWS IAM](https://aws.amazon.com/iam/) the accounts/user profiles in AWS IAM Identity Center are completely separate from accounts in AWS IAM.**
+
+1. Login to the AWS console and navigate to the IAM Identity Center service
+1. In the left nav, click "Users"
+1. For each user you'd like to grant access to viewing metrics, click the "Add user" button and follow the workflow
+1. Follow instructions [here](https://docs.aws.amazon.com/grafana/latest/userguide/AMG-manage-users-and-groups-AMG.html) for adding granting access for those users to your Grafana workspace
+
+#### Configure dashboard
+
+Once you have an Identity Center user with permissions to administer your Grafana workspace, it's time to configure the workspace dashboard. From the Grafana workspace page in the AWS console, click the link under "Grafana workspace URL". After signing in this will take you to your Grafana workspace. 
+
+To enable viewing metrics, add the Prometheus server as a data source for your Grafana workspace: 
+
+1. Click the AWS logo in the left nav
+1. Click "Amazon Managed Service for Prometheus"
+1. Add the data source for Prometheus
+
+With Prometheus connected as a Grafana workspace, panels can now be created in Grafana that display metrics from the CiviForm server. There are many metrics available, and many ways to display them. You can get started with the some basic metrics by importing a pre-built CiviForm dashboard:
+
+1. Hover over the '+' icon in the left nav
+1. Click the 'Import' option 
+1. Paste the following JSON into the "Import via panel JSON" and click "Load"
+
+```json
```

Comment:
Is there an easy way to add this file to the repo somewhere then make this a file download?

Reply:
Probably. The reader needs to paste the actual text into the UI though, so making them download the file first adds extra steps for downloading, opening it in an editor. Thoughts?

Reply:
Switched to linking to the raw JSON content in a github gist